docs/scaling/multi-node-setup.txt

```diff
@@ -48,6 +48,23 @@ If the node name is omitted, then it's generated dynamically on startup.
 
 .. _inter-node-comms:
 
+Node Type
+---------
+A Node can have various roles in the cluster:
+
+**Master-eligible node**
+
+A node that has node.master set to true, which makes it eligible to
+be elected as the master node, as part of the :ref:`master-node-election`.
+
+**Data node**
+
+A node that has node.data set to true. Data nodes hold soley data.
+
+A node can have multiple roles at the same time. `More information on Node types
+<https://crate.io/docs/crate/reference/en/latest/config/node.html#node-types>`_
+can be found in CrateDB's reference guide.
+
```

Comment:
I'm missing the `coordinating-only` node. At least it is documented to use that role here https://github.com/crate/guide/pull/138/files#diff-886b6c19b61db935d205fea0236a52abR78